docs/content/how-to/aws/create-aws-hosted-cluster-autonode.md

```diff
@@ -0,0 +1,452 @@
+---
+title: Create AutoNode hosted clusters powered by Karpenter
+---
+
+HyperShift AutoNode (Powered by Karpenter) is ...(TODO)
+
+To create a hosted cluster with autoNode enabled the Hypershift Operator
+needs to be installed with the `--tech-preview-no-upgrade=true` flag.
+
+The Hypershift controller creates and manages the default [`EC2NodeClass`][ec2nodeclass]
+in the hosted cluster allowing you to deploy workloads based in
+your environment with [`NodePools`][nodepools].
```

Comment:
let's clarify nodepools.karpenter.sh so is not ambiguous with nodepool.hypershift.openshift.io